tests/testthat/test-tidyeval.R

```diff
@@ -0,0 +1,11 @@
+context("tidyeval")
+
+test_that("bang bang", {
+  expect_warning(test_collection("tidyeval", "bang_bang",
+                                transformer = style_text), NA)
+})
+
+test_that("bang bang", {
```

Comment:
you mean to something more verbose like "spaces are removed in bang-bang operator"?